### ISSUE TITLE
database/gdb: Have you ever considered using gorm directly instead of maintaining a separate orm in a framework?

### ISSUE THREAD

**Issue by wisonlau** [gogf/gf#3954]
### Is your feature request related to a problem?

Option Yes

### Describe the solution you'd like

框架只需提供解耦方案就可以

### Describe alternatives you've considered

框架只需提供解耦方案就可以

**Comment by gqcn** [comment]
Yes, please refer to https://goframe.org/docs/design/project-dao-pain

**Comment by nuan1989** [comment]
感觉gdb更好用, 哈哈哈.
